src/sentry/static/sentry/app/views/settings/components/settingsNavigationGroup.tsx

```diff
@@ -24,6 +25,20 @@ const SettingsNavigationGroup = (props: NavigationGroupProps) => {
           ...(project ? {projectId: project.slug} : {}),
         });
 
+        const handleClick = () => {
+          //only call the analytics event if the URL is changing
+          if (recordAnalytics && to !== window.location.pathname) {
+            trackAnalyticsEvent({
```

Comment:
This will generate a warning if `project` is `undefined`, but we will also get a warning for the `organization_id` being a number instead of a string because it's coerced later so I figured no big deal.